src/icons/general/IconSecurityClassification.js

```diff
@@ -0,0 +1,44 @@
+// @flow
+import * as React from 'react';
+
+import AccessibleSVG from '../accessible-svg';
+import { bdlGray } from '../../styles/variables';
+
+type Props = {
+    className?: string,
+    color?: string,
+    height?: number,
+    strokeWidth?: number,
+    /** A text-only string describing the icon if it's not purely decorative for accessibility */
+    title?: string | React.Element<any>,
+    width?: number,
+};
+
+const IconSecurityClassification = ({
+    className = '',
+    height = 13,
+    color = bdlGray,
+    title,
+    strokeWidth = 1,
+    width = 11,
+}: Props) => (
+    <AccessibleSVG
+        className={`bdl-IconSecurityClassification ${className}`}
```

Comment:
I recommend using `classnames` instead, to prevent the addition of extra whitespace.

Reply:
Hey @sahiga , what extra whitespace are you referring here ?

Reply:
@feixuanli In the event that this component doesn't have a `className` prop, the computed class name on the SVG will be "bdl-IconSecurityClassfication " instead of "bdl-IconSecurityClassification." Other components in BUIE use `classnames` to avoid this issue. This is pretty minor, though.

Reply:
Thanks, updated.